proxy/proxy/httpserver.py

```diff
@@ -0,0 +1,171 @@
+"""proxy server"""
+from http import HTTPStatus
+from http.server import ThreadingHTTPServer, BaseHTTPRequestHandler
+import json
+import logging
+import os
+import ssl
+import zlib
+import requests
+
+logging.basicConfig(level=logging.DEBUG)
+
+HOST = "127.0.0.1"
+PORT = 8443
```

Comment:
I see that we are configuring the port through several places. I assume we will need to use environment variables here too.